src/isomorphic/modern/element/__tests__/ReactJSXElementValidator-test.js

```diff
@@ -109,6 +109,19 @@ describe('ReactJSXElementValidator', () => {
     );
   });
 
+  it('does not warns for fragments of multiple elements without keys', () => {
+    spyOn(console, 'error');
+
+    void (
```

Comment:
Let's actually render these elements. The missing key warning is fired during reconciliation, not element creation.